examples/02_text_with_string_encoders.py

```diff
@@ -301,13 +301,23 @@ def plot_performance_tradeoff(results):
         ax.set_ylabel("ROC AUC")
         ax.set_title("Prediction performance / training time trade-off")
 
+    log_xlim = np.log10(ax.get_xlim())
+    x_log_range = log_xlim[1] - log_xlim[0]
+    y_range = ax.get_ylim()[1] - ax.get_ylim()[0]
+    arrow_tip_x_log = log_xlim[0] + 0.05 * x_log_range
+    arrow_tip_y = ax.get_ylim()[1] - 0.05 * y_range
+    arrow_start_x_log = arrow_tip_x_log + 0.04 * x_log_range
+    arrow_start_y = arrow_tip_y - 0.2 * y_range
+    arrow_tip_x = np.power(10, arrow_tip_x_log)
+    arrow_start_x = np.power(10, arrow_start_x_log)
```

Comment:
Ouch.

This adds a lot of chunk of code to an example for cosmetic reasons.

I don't like that. More code like this is frightening for users. Examples should strive to have less code, not more code